doc/user/layouts/partials/create-source/connector/s3/with-options.html

```diff
@@ -21,7 +21,7 @@
 | Key Name Source       | Permissions Required                                                                     |
 |-----------------------|------------------------------------------------------------------------------------------|
 | All                   | [`GetObject` permission][getobject] for the objects that they will download              |
-| **BUCKET SCAN**       | [`ListObject` permission][listobject] for the buckets it will scan                       |
+| **BUCKET SCAN**       | [`ListObject` permission][listobject] for the buckets it will scan, **unless** the `MATCHING` pattern can only list a single object, in which cas materialize will only perform the necessary `GetObject` API call.                        |
```

Comment:
typo: "cas" -> "case"

Also, Materialize needs to be capitalized in the docs! Always always, unless you are referring to `materialized` the binary.

And minor nit: "unless the `MATCHING` pattern can only match a single object" is clearer I think; "list" isn't really a verb that can apply to a pattern but "match" of course is.